pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSchemas.java

```diff
@@ -83,4 +88,44 @@ void run() throws Exception {
         }
     }
 
+    @Parameters(commandDescription = "Provide the schema via a topic")
+    private class PojoSchema extends CliCommand {
+        @Parameter(description = "persistent://tenant/namespace/topic", required = true)
+        private java.util.List<String> params;
+
+        @Parameter(names = { "-j", "--jar" }, description = "jar filepath", required = true)
+        private String jarFilePath;
+
+        @Parameter(names = { "-t", "--type" }, description = "type avro or json", required = true)
+        private String type;
+
+        @Parameter(names = { "-c", "--class-name" }, description = "class name of pojo", required = true)
```

Comment:
`--class-name` to `--classname`. `--classname` is more common than `--class-name`.